File: LeetCode/001295.py
from typing import List


class Solution:
    def get_digit_count(self, n: int) -> int:
        result = 0
        temp = n
        while True:
            result += 1
            temp = temp // 10
            if temp == 0:
                break
        return result

    def findNumbers(self, nums: List[int]) -> int:
        result = 0
        for n in nums:
            if self.get_digit_count(n) % 2 == 0:
                result += 1
        return result

File: LeetCode/001295.go
package leetcode

func getDigitCount(n int) int {
	count := 0
	for {
		div := n / 10
		count += 1
		if div <= 0 {
			break
		} else {
			n = div
		}
	}
	return count
}

func findNumbers(nums []int) int {
	result := 0
	for _, v := range nums {
		digitCount := getDigitCount(v)
		if digitCount%2 == 0 {
			result += 1
		}
	}
	return result
}
